test/test_listener.cc

```diff
@@ -20,7 +20,9 @@ void TestListener::OnTestEnd(const ::testing::TestInfo& test_info) {
                    absl::StrCat("MainThreadLeak: [", test_info.test_suite_name(), ".",
                                 test_info.name(), "] test exited before main thread shut down"));
   }
-  Runtime::RuntimeFeaturesDefaults::get().restoreDefaults();
+  // Reset flags, and latch the defaults for next test.
+  saver_.reset();
```

Comment:
Presumably this like need to happen before constructing the new Saver? (Otherwise we could just remove this line.) If so, mind adding a comment?

Reply:
I thought I did - I'll add more context

Reply:
Thanks this is clear now!